docsets/FoundationEmail/docset.json

```diff
@@ -0,0 +1 @@
+{"author":{"name":"Joe Workman","link":"https://twitter.com/joeworkman"},"aliases":["Foundation for Email","Foundation","Email","Inky"],"archive":"FoundationEmail.tgz","version":"1.0.0","name":"Foundation for Email","specific_versions":[{"archive":"versions/1.0.0/FoundationEmail.tgz","version":"1.0.0"}]}
```

Comment:
The versions should be the versions of the docs, not the docset itself. Foundation for Emails is at v2 as far as I can tell.

Also please rename to "Foundation for Emails".